app/code/core/Mage/ImportExport/Model/Resource/Helper/Mysql4.php

```diff
@@ -49,11 +56,26 @@ public function getMaxDataSize()
     public function getNextAutoincrement($tableName)
     {
         $adapter = $this->_getReadAdapter();
+        try {
+            $this->setInformationSchemaStatsExpiry();
+        } catch (Exception $e) {
+            //this will throw an exception for < mysql8
+        }
```

Comment:
```suggestion
        $this->setInformationSchemaStatsExpiry();
```

Reply:
@colinmollenhour exactly, I just sent the suggestion for the other part of the code ;-)